tests/pom.xml

```diff
@@ -565,25 +593,6 @@
         <hono.jdbc.db.registry.password>${hono.jdbc.db.admin.password}</hono.jdbc.db.registry.password>
       </properties>
     </profile>
-    <profile>
-      <id>device-registry-file</id>
```

Comment:
any reason why we need to remove this profile as part of this PR?

Reply:
seems a mistake. I'd return the device-registry-file profile back - device-registry-file